moulinette/interfaces/api.py

```diff
@@ -272,13 +272,14 @@ def setup(self, app):
             name="login",
             method="POST",
             callback=self.login,
-            skip=["actionsmap"],
+            skip=[filter_csrf, "actionsmap"],
```

Comment:
I'm kinda confused ... I understand that basic, "raw" HTML form won't include the special X-Requested-With header ... but it looks like disabling this checks opens up the very security issue that this CSRF is supposed to prevent ... or would a CSRF token help fix the "raw" HTML form support, rather than relying only on X-Requested-With ?

Reply:
My understanding: CSRF is a problem where you steal credentials from an authenticated user from another webpage. On the login/logout route, it should not have any security implications.

Reply:
Alrighty:

>Definition. Cross-Site Request Forgery (CSRF) is an attack that forces **authenticated** users to submit a request to a Web application **against which they are currently authenticated**.